app/controllers/miq_ae_customization_controller.rb

```diff
@@ -67,6 +67,17 @@ def upload_import_file
 
   def import_service_dialogs
     if params[:commit] == _('Commit')
+
+      if params[:dialogs_to_import].blank?
+        add_flash(_("At least one Service Dialog must be selected."), :error)
+        render :update do |page|
+          page << javascript_prologue
+          page.replace("flash_msg_div", :partial => "layouts/flash_msg")
+          page << "miqSparkle(false);"
+        end
+        return
+      end
```

Comment:
Also. If this is an explorer screen, you might need `render_flash_and_scroll`

if there are too many items on the screen, the flash might be scrolled off the `main_div` 's visible area